apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-applications/vo-applications.component.ts

```diff
@@ -136,14 +136,15 @@ export class VoApplicationsComponent implements OnInit {
         ) {
           this.configuredColumns = attribute.value as Array<string>;
           this.configuredFedColumns = this.configuredColumns.filter((column) =>
-            this.fedAttrNames.includes(column)
+            this.fedAttrs.some((attr) => attr.friendlyName === column)
           );
         } else {
           this.configuredColumns = [];
           this.configuredFedColumns = [];
         }
         this.columnsAuth = attribute.writable;
         this.currentColumns = this.refreshColumns();
+        this.loading$ = of(false);
```

Comment:
Step 2: remove this line:
```suggestion
```
Now I think it works on init/refresh/change and also for vo and for a group. After these changes the behavior seems the same as the behavior of our devel.